bin/process

```diff
@@ -120,18 +115,18 @@ function verify1() {
     for batch_id in $(jq -r '.[].batch_id' "${DATA_CONFIG}"); do
         prefix="batch_id=${batch_id}"
         n_data=$(config_get "${batch_id}" "n_data")
-        if ! gsutil -q stat "${input}/${prefix}/*"; then
+        if ! mc stat "internal/${input}/${prefix}" &> /dev/null; then
             echo "data for ${batch_id} does not exist, skipping..."
             continue
         fi
         spark_submit verify1 \
             --batch-id "${batch_id}" \
             --n-data "${n_data}" \
-            --input "${input}/${prefix}" \
-            --output "${output_internal}/${prefix}"
+            --input "s3a://${input}/${prefix}" \
```

Comment:
We use the s3a connector for spark, which is the reason that the bucket is no longer prefixed. It works surprisingly well.